app/src/main/java/com/example/imageEditor2/Api.kt

```diff
@@ -17,18 +19,23 @@ interface Api {
 
     @GET
     suspend fun searchPhotos(
-        @Query("page") page: String,
-        @Query("query") querySearch: String,
+        @Query(PAGE) page: String,
+        @Query(QUERY_SEARCH) querySearch: String,
     )
 
     @POST("photos/{id}/like")
     suspend fun likeImage(
         @Path("id") id: String,
     )
 
-    @DELETE
-    suspend fun dislikeImage()
+    @DELETE("photos/{id}/like")
+    suspend fun dislikeImage(
+        @Path("id") id: String,
```

Comment:
```suggestion
        @Path(ID) id: String,
```
ca này cũng common thường hay gặp define luôn e nhé!